index.html

```diff
@@ -51,7 +50,7 @@
     </header>
     <section class="main">
       <div class="heading">
-        <div class="principal">
+        <div class="princip">
```

Comment:
- [ OPTIONAL ] In your speakers section I think it would be great if you make the section items aligned as shown in the given design since I can see it is very different in your design but please work on it to have it the same 💯 and I am assiging as optional because the previous reviewer didn't mention it but it is very important to fix so please work on it 🚀🔥😊
## Please fix the styles acoording to the alignment and the gaps between the img and text 🔥🔥🔥


- The image should aligned beside the text as row
- The heading and paragraph should be in column 

- Your design
![image](https://github.com/aradradev/easyBankTC/assets/58553711/194524d9-9fef-4adf-9573-96e3f18674d0)


- Given
![image](https://github.com/aradradev/easyBankTC/assets/58553711/a72668f9-2d55-4e55-84ff-af8de034c0d8)